src/compiler/scanner.rs

```diff
@@ -1,5 +1,44 @@
+#![allow(non_upper_case_globals)]
+
+use std::array::IntoIter;
+use std::collections::HashMap;
+use std::iter::FromIterator;
+
 use crate::{CharacterCodes, SyntaxKind};
 
+lazy_static! {
+    static ref text_to_keyword_obj: HashMap<&'static str, SyntaxKind> =
+        HashMap::from_iter(IntoIter::new([
```

Comment:
I think that if you put a `&` in front of this array, it'd be iterable, so you could use the more normal `into_iter` or whatever here.

Reply:
I'm not sure exactly what that would look like, this was mimicking some answer re how to approximate a `HashMap` literal in Rust